pkg/coredata/postgres/0001_init.down.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE action_versions;
```

Comment:
I think this file needs to be dropped into the migrations dir?

Reply:
Hmm, it looks like goose separates things by up && down, not by files.  Maybe this can be removed?

Reply:
Ah you're right - I thought I deleted this outdated file - should be gone as of latest commit!